src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,122 @@
+import { useSearchParams } from 'react-router-dom';
+import { Sex } from '../types';
+import cn from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+
+  const handleQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('query', event.target.value);
+    setSearchParams(params);
+  };
+
+  const handleGenderChange = (
+    gender: Sex,
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    event.preventDefault();
+    const params = new URLSearchParams(searchParams);
+
+    params.set('sex', gender);
+    setSearchParams(params);
+  };
```

Comment:
and for sex as well as for query - empty parameter should not be in search params